### DISCUSSION TITLE
Any way to config with css?

### DISCUSSION THREAD

**Discussion by sevenc-nanashi** [tonsky/FiraCode#1235] [Q&A] [answered]
I'm using fira code with Code Injector to GitHub, and how do I enable ss01(or something) with css?

**Comment by wopian** [comment, marked as answer]
Use `font-feature-settings`: https://developer.mozilla.org/en-US/docs/Web/CSS/font-feature-settings

```css
font-feature-settings: 'ss01', 'ss02', 'ss03';
```

**Reply by sevenc-nanashi** [reply]
thanks!
